fe1-web/src/features/rollCall/components/__tests__/EventRollCall.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import STRINGS from 'resources/strings';
+import { Hash, Timestamp } from 'core/objects';
+import { useNavigation } from '@react-navigation/native';
+
+import * as reactRedux from 'react-redux';
+import { mockLao } from '__tests__/utils/TestUtils';
+import { requestOpenRollCall, requestReopenRollCall } from 'features/rollCall/network';
+import EventRollCall from '../EventRollCall';
+import { EventTypeRollCall, RollCall, RollCallStatus } from '../../objects';
+
+const ID = new Hash('rollCallId');
+const NAME = 'myRollCall';
+const LOCATION = 'location';
+const TIMESTAMP_START = new Timestamp(1620255600);
+const TIMESTAMP_END = new Timestamp(1620357600);
+const ATTENDEES = ['attendee1', 'attendee2'];
+const rollCallState: any = (mockStatus: RollCallStatus) => {
+  return {
+    id: ID.valueOf(),
+    eventType: EventTypeRollCall,
+    start: TIMESTAMP_START.valueOf(),
+    name: NAME,
+    location: LOCATION,
+    creation: TIMESTAMP_START.valueOf(),
+    proposedStart: TIMESTAMP_START.valueOf(),
+    proposedEnd: TIMESTAMP_END.valueOf(),
+    status: mockStatus,
+    attendees: ATTENDEES,
+    idAlias: mockStatus === RollCallStatus.CREATED ? undefined : ID.valueOf(),
+  };
+};
+
+const mockRollCallClosed = RollCall.fromState(rollCallState(RollCallStatus.CLOSED));
+const mockRollCallCreated = RollCall.fromState(rollCallState(RollCallStatus.CREATED));
+const mockRollCallOpened = RollCall.fromState(rollCallState(RollCallStatus.OPENED));
+
+const mockRenderRollCall = (rollCall: RollCall, isOrganizer: boolean) => {
+  return render(<EventRollCall event={rollCall} isOrganizer={isOrganizer} />);
+};
```

Comment:
I'd honestly delete this. I don't think it is clearer and the actual render function isn't that long either.